src/lib/loaders/ConfigLoader.ts

```diff
@@ -0,0 +1,15 @@
+import {CodeAnalyzerConfig} from '@salesforce/code-analyzer-core';
+
+export interface ConfigLoader {
+	loadConfig(configPath?: string): CodeAnalyzerConfig;
+}
+
+export class ConfigLoaderImpl implements ConfigLoader {
+	public loadConfig(configPath?: string): CodeAnalyzerConfig {
+		if (configPath) {
+			return CodeAnalyzerConfig.fromFile(configPath);
```

Comment:
Was technically saving this part for a later story, but it turned out to be really easy to implement, so I just went ahead and did it.